modules/cli/pages/index.adoc

```diff
@@ -6,6 +6,8 @@ A Command Line Interface tool (CLI) is used to provision data extracted from Bon
 
 The command line tool is delivered as a package, containing a basic documentation about commands and a sample configuration file.
 
+You can download the tool here:
```

Comment:
`here` should be more accurate: CSC ? website ? github ?

Reply:
Currently, the storage of the tool is not validated, so I suggested contacting Bonita to recover the tool (which we are currently doing). The subject is taken into account (CSC or PI APP)